### ISSUE TITLE
Option "Dons"

### ISSUE THREAD

**Issue by Havra-Esperantista-Grupo** [Havra-Esperantista-Grupo/HEG#7]
- Un de nos membres tient à ce qu'on propose à nos visiteurs une possibilité de faire un "Don" à notre Association. On peut toujours rêver mais, en tout cas, est-ce envisageable ?

**Comment by axel584** [comment]
Oui, ça peut même être assez facile avec un bouton "paypal" (qui s'occupe de gérer la carte bleue et compagnie)
